common/djangoapps/student/migrations/0042_grant_sales_admin_roles.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models, IntegrityError
+
+
+class Migration(DataMigration):
```

Comment:
@wedaly This is the new data migration we discussed
@e0d  FYI I'm going to use this to migrate finance_admin roles to sales_admin roles.